components/edit-collective/Menu.js

```diff
@@ -168,7 +168,7 @@ const sectionsDisplayConditions = {
   [EDIT_COLLECTIVE_SECTIONS.HOST]: c => isCollective(c),
   [EDIT_COLLECTIVE_SECTIONS.MEMBERS]: c => isOneOfTypes(c, CollectiveType.COLLECTIVE, CollectiveType.ORGANIZATION),
   [EDIT_COLLECTIVE_SECTIONS.PAYMENT_METHODS]: c =>
-    isOneOfTypes(c, CollectiveType.COLLECTIVE, CollectiveType.ORGANIZATION) && !isFund(c),
+    isOneOfTypes(c, CollectiveType.COLLECTIVE, CollectiveType.ORGANIZATION, CollectiveType.USER) && !isFund(c),
```

Comment:
Should not be there, was mean to go in https://github.com/opencollective/opencollective-frontend/pull/4436